src/client/components/pages/MealByIdInfo.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+export function MealByIdInfo({ title, description, location, price, when }) {
+    return (
+      <>
+        <img src="https://via.placeholder.com/450x370"></img>
```

Comment:
instead of image placeholders, you can upload some beautiful photos. to make app beautiful.